src/main/scala/tile/Core.scala

```diff
@@ -22,9 +22,12 @@ trait CoreParams {
   val useAtomics: Boolean
   val useAtomicsOnlyForIO: Boolean
   val useCompressed: Boolean
-  val useBitManip: Boolean = false
+  val useBitManip: Boolean
+  val useBitManipCrypto: Boolean
```

Comment:
Does anything else in `BaseTile/BaseCore` use `useBitManipCrypto/useCryptoNIST/useCryptoSM`? maybe those should be rocket-core only params

Reply:
> maybe those should be rocket-core only params

useABLU is a rocket only param, but these params are not. They correspond to ratified isa extensions and could be reused by other cores instead of each core defining their own parameters.

Also, making these params private to rocket params will cause `isaDTS` not able to find corresponding options (unless with type cast).

I do know this will cause pain for other cores to migrate to new CoreParams. One solution is to provide default value and override them in RocketCoreParams.

Reply:
Ok in a past commit, isaDTS did not read these params. You have fixed this now.

The migration burden is small here, I think this is currently ok.